devicetypes/smartthings/ecobee-sensor.src/ecobee-sensor.groovy

```diff
@@ -68,6 +67,6 @@ def refresh() {
 
 void poll() {
 	log.debug "Executing 'poll' using parent SmartApp"
-	parent.pollChild(this)
+	parent.pollChild()
```

Comment:
Just because I'm new.  Why aren't you passing the device type to poll anymore?  What kind of effect does that have?